common/utils/api-utils.js

```diff
@@ -16,22 +16,22 @@ export const get = async (path, { token } = {}) =>
     headers: setAuthorizationHeader(token),
   });
 
-export const post = async (path, body) => {
-  const result = await OperationCodeAPI.post(`/${path}`, body, {
-    headers: setAuthorizationHeader(),
+/**
+ * @param {string} path
+ * @param {{any}} body
+ * @param {{noAuth?: boolean}}options
```

Comment:
you'll wanna add a space between } and options

Reply:
Man I would have sworn I fixed that.